actors/builtin/paych/paych_test.go

```diff
@@ -45,63 +45,81 @@ func TestPaymentChannelActor_Constructor(t *testing.T) {
 		actor.constructAndVerify(t, rt, payerAddr, payeeAddr)
 	})
 
-	nonAccountCodeID := builtin.MultisigActorCodeID
+	t.Run("creates a payment channel actor after resolving non-ID addresses to ID addresses", func(t *testing.T) {
+		payerAddr := tutil.NewIDAddr(t, 101)
+		payerNonId := tutil.NewBLSAddr(t, 102)
+
+		payeeAddr := tutil.NewIDAddr(t, 103)
+		payeeNonId := tutil.NewBLSAddr(t, 104)
+
+		builder := mock.NewBuilder(ctx, paychAddr).
+			WithCaller(callerAddr, builtin.InitActorCodeID).
+			WithActorType(payerAddr, builtin.AccountActorCodeID).
+			WithActorType(payeeAddr, builtin.AccountActorCodeID)
+		rt := builder.Build(t)
+		rt.AddIDAddress(payerNonId, payerAddr)
+		rt.AddIDAddress(payeeNonId, payeeAddr)
+
+		actor.constructAndVerify(t, rt, payerNonId, payeeNonId)
+	})
+
 	testCases := []struct {
-		desc        string
-		fromCode    cid.Cid
-		fromAddr    addr.Address
-		toCode      cid.Cid
-		toAddr      addr.Address
-		expExitCode exitcode.ExitCode
+		desc               string
+		paymentChannelAddr addr.Address
+		callerCode         cid.Cid
+		newActorCode       cid.Cid
```

Comment:
These tests were already not great, but I don't understand these changes. The `paymentChannelAddr` is always the same and `newActorCode`, the code of the payment channel, must be `PaymentChannelActorCodeID`, not whatever is set here.

These tests should probably exercise the cases where the address was not initially resolved, and hence created. I think all the original cases are still relevant, though now with different expected outcomes.